var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -94,16 +94,15 @@ class Sirius(CMakePackage, CudaPackage):
     depends_on('magma', when='+magma')
     depends_on('boost cxxstd=14 +filesystem', when='+boost_filesystem')
 
-    depends_on('spfft@1.0.3:', when='@6.4.0:')
-    depends_on('spfft+cuda', when='@6.4.0:+cuda')
-    depends_on('spfft+rocm', when='@6.4.0:+rocm')
+    depends_on('spfft+mpi', when='@6.4.0:')
+    depends_on('spfft+cuda', when='+cuda ^spfft')
+    depends_on('spfft+rocm', when='+rocm ^spfft')
+    depends_on('spfft+openmp', when='+openmp ^spfft')
 
-    depends_on('spla@1.4.0:', when='@7.0.0:')
-    depends_on('spla+cuda', when='@7.0.0:+cuda')
-    depends_on('spla+rocm', when='@7.0.0:+rocm')
-
-    depends_on('elpa+openmp', when='+elpa+openmp')
-    depends_on('elpa~openmp', when='+elpa~openmp')
+    depends_on('spla', when='@7.0.0:')
```

Comment:
The same here, 1.1.0 is required for the latest release and possibly 1.4.0 in the future